applications/Solang_Playground.md

```diff
@@ -1,8 +1,10 @@
 # Solang Playground
 
 
-- **Team Name:** Salaheldin Soliman
-- **Payment Address:** 0x9036b20920ddf473e796d24afd0fe04d4dde0b68
+- **Team Name:** Salaheldin Soliman and Tarek Elsayed
+- **Payment Address:** 
```

Comment:
You added another payment address. Does this mean you both want to receive 50% of the total amount?

Reply:
Nope, but after each milestone, we will file two invoices (one per each contributer) based on our agreement.
If the exact split is to be amended to this file that should be better of course

Reply:
Got it, thanks for clarifying.